src/chatd.cpp

```diff
@@ -1892,7 +1874,7 @@ void Connection::execCommand(const StaticBuffer& buf)
                 if (mChatdClient.mRtcHandler)
                 {
                     auto& chat = mChatdClient.chats(chatid);
-                    if (!chat.isGroup() || (chat.isGroup() && mChatdClient.mKarereClient->areGroupCallsEnabled()))
+                    if (!chat.isGroup())
```

Comment:
I think we have to remove this condition too. We have to manage CALLTIME for 1to1 and group chatrooms

Reply:
You are correct, good catch. Please, can you change it, @alber2510 ?

Reply:
ok